S7.Net/Conversion.cs

```diff
@@ -151,6 +151,31 @@ public static bool SelectBit(this byte data, int bitPosition)
             return (result != 0);
         }
 
+        /// <summary>
+        /// Helper to set a bit value to the given byte at the bit index.
+        /// </summary>
+        /// <param name="data"></param>
+        /// <param name="bitPosition"></param>
+        /// <param name="value"></param>
+        public static void SetBit(this ref byte data, int bitPosition, bool value)
```

Comment:
```suggestion
        public static void SetBit(this ref byte data, int index, bool value)
```

`bitPosition` is described as `index` in the method summary, I believe `index` is also what the exact name of the parameter should be.

Reply:
😂I was wanted to add these infos and use `index` instead of `bitPosition`, but when I saw `SelectBit` method and it's summary, to keep the same style and format, I wrote that codes.

I have updated with your suggested changes and modified the `SelectBit` method to keep the same style.